### PR TITLE
feat(tracing): skeleton and reactivity fix

### PR DESCRIPTION
**Pull request by sumimakito**
# Summary

This pull request adds a new property `skeleton` to control the skeleton state of `TraceViewerPage` and `WaterfallView`, and fixes a potential reactivity issue in `WaterfallView`.

Screenshot 2024-12-03 at 16 08 39

### CODE REVIEW

File: packages/core/tracing/sandbox/pages/TraceViewerPage.vue [hunk 1]
```diff
@@ -96,3 +118,14 @@ const url = `https://example.com${path}`
...
+<style lang="scss" scoped>
+.controls {
+  z-index: 10000;
+  bottom: 16px;
+  left: 16px;
+  margin-bottom: 16px;
+  position: fixed;
+  width: auto;
+}
+</style>
```
Comment (Leopoldthecoder) [inline packages/core/tracing/sandbox/pages/TraceViewerPage.vue:131]: Do we need a new `<style>` block here?

File: packages/core/tracing/src/components/trace-viewer/TraceViewer.vue [hunk 2]
```diff
@@ -85,8 +105,9 @@ const { i18n: { t } } = composables.useI18n()
 
 const props = defineProps<{
   config: TraceViewerConfig
-  rootSpan: SpanNode
+  rootSpan?: SpanNode
   url?: string
+  skeleton?: boolean
```
Comment (Leopoldthecoder) [inline packages/core/tracing/src/components/trace-viewer/TraceViewer.vue:110]: Could we rename this prop to `showSkeleton` or `isLoading`?

File: packages/core/tracing/src/components/waterfall/WaterfallView.vue [hunk 3]
```diff
@@ -95,6 +107,10 @@ const props = defineProps({
     type: Object as PropType<SpanNode>,
     default: () => undefined,
   },
+  skeleton: {
```
Comment (Leopoldthecoder) [inline packages/core/tracing/src/components/waterfall/WaterfallView.vue:110]: `showSkeleton` or `isLoading`

File: packages/core/tracing/src/components/trace-viewer/TraceViewer.vue (hunk 2 above)
Comment (sumimakito) [reply packages/core/tracing/src/components/trace-viewer/TraceViewer.vue]: Good idea. Updated

File: packages/core/tracing/sandbox/pages/TraceViewerPage.vue (hunk 1 above)
Comment (sumimakito) [reply packages/core/tracing/sandbox/pages/TraceViewerPage.vue:129]: Oops. Removed

**Review by Leopoldthecoder — APPROVED** [review]
LGTM

**Pull request merged**